src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/client/payhub/response/PaymentResult.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.client.payhub.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class PaymentResult {
+
+    public final List<String> paymentDcns;
+
+    public PaymentResult(
+        @JsonProperty("payment_dcns") List<String> paymentDcns
```

Comment:
I have just emailed F&P team because (as per swagger specs this is correct) this doesn't look correct to me. As per our initial discussion they agreed to return unique id for each request.

Reply:
we can add later when they confirm . ??